telegram/_bot.py

```diff
@@ -3246,7 +3280,11 @@ async def edit_message_caption(
                 0-:tg-const:`telegram.constants.MessageLimit.CAPTION_LENGTH` characters after
                 entities parsing.
             parse_mode (:obj:`str`, optional): |parse_mode|
-            caption_entities (List[:class:`telegram.MessageEntity`], optional): |caption_entities|
+            caption_entities (Sequence[:class:`telegram.MessageEntity`], optional):
+                |caption_entities|
```

Comment:
Does the text of this substitution need adaption? you edited "List" to "Sequence"e.g. in line 3206, maybe this is needed here as well? If `|caption_enitites|` is used in docstrings of attributes as well, then we'll probably need a second substitution that says "Tuple" …